api/MediaDevices.json

```diff
@@ -207,9 +223,178 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/getUserMedia",
           "support": {
-            "webview_android": {
+            "chrome": [
+              {
+                "version_added": "52"
+              },
+              {
+                "version_added": "47",
+                "version_removed": "52",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": [
+                  "Older versions of Chrome implement <code>navigator.webkitGetUserMedia</code>, a prefixed form of the legacy <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia'><code>navigator.getUserMedia</code></a> API."
+                ]
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "52"
+              },
+              {
+                "version_added": "47",
+                "version_removed": "52",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features",
```

Comment:
Like before, this should be `"name": "Experimental Web Platform features",` and `"value_to_set": "Enabled"` to more closely reflect the UI for this

Reply:
Regarding ```"value_to_set"```, I also had the same thought but just used "true" based on [schema definition](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#user-content-flags).

Reply:
Oh good catch! I think that will need some attention on separate issue/PR.